setup.py

```diff
@@ -246,7 +246,7 @@ def check_submodules():
         check_submodules()
         ext_modules.extend([
             CMakeExtension("intel_extension_for_transformers.neural_engine_py", "intel_extension_for_transformers/llm/runtime/deprecated/"),
-            CMakeExtension("intel_extension_for_transformers.llm.runtime.graph.Model", "intel_extension_for_transformers/llm/runtime/graph/"),
+            CMakeExtension("intel_extension_for_transformers.llm.runtime.graph.mpt_cpp", "intel_extension_for_transformers/llm/runtime/graph/"),
```

Comment:
When running the `python setup.py develop' command, an error will appear, unable to copy Model_xxx.so, To fix this problem, just use an existing object.

After repairing, there is still a problem, because qbit has similar issues, but which can't be solved by my solution.

Reply:
I think the recommended way is to ```pip install -v .```, how about this?

Reply:
both works